### PR TITLE
Automatic RPC updates every 6 hours!!!!

### LINKED ISSUE #5: RPC auto updating
Rather than having it update on restart, since that won't happen a whole lot, we can just do it every once in a while. SetInterval!

### PR DESCRIPTION
**Pull request by PlerionPlorion**

### CODE REVIEW

File: src/events/ready.js [hunk 1]
```diff
@@ -30,22 +27,27 @@ module.exports = {
...
-            client.user.setPresence({
-                activities: [
-                    {
-                        name: "against " + rolesList[randomIndex],
-                        type: 0, // enum: 0 = playing, 1 = streaming, 2 = listening, 3 = watching, 4 = custom, 5 = competing
-                        status: "online",
-                        afk: false,
-                    },
-                ],
+            process.on("SIGINT", () => {
```
Comment (GalexY727) [inline src/events/ready.js:68]: Can you add a comment on what this is just cuz i'm a little lost lol

File: src/events/ready.js [hunk 2]
```diff
@@ -30,22 +27,27 @@ module.exports = {
...
+                clearInterval(intervalId);
             });
-        } else {
-            console.log("No roles found.");
         }
         // console.log("buh");
```
Comment (GalexY727) [inline src/events/ready.js:73]: THE BUHHHH

File: src/events/ready.js [hunk 3]
```diff
@@ -9,18 +9,15 @@ module.exports = {
     once: true,
     async execute(client) {
         console.log(`Ready! Logged in as ${client.user.tag}`);
-
+        let intervalId;
         let rolesList = [];
 
         for await (const guild of client.guilds.cache.values()) {
             try {
                 // Iterate over each role in the guild
```
Comment (GalexY727) [inline src/events/ready.js:23]: we should delete this while we're at it

**Review by GalexY727 — APPROVED** [review]
Looks nice! Should we have a guild id check so that it only looks at frcsd's teams?

**Comment by PlerionPlorion** [comment]
Resolved comments and made reactions slightly rarer by having more randomness

File: src/events/funStuff.js [hunk 4]
```diff
@@ -42,7 +42,7 @@ module.exports = {
                 `(?:^|[^a-zA-Z0-9])${keyword}(?:$|[^a-zA-Z0-9])`,
                 "i"
             );
-            if (Math.random() < 0.5 && regex.test(message.content)) {
+            if (Math.random() < Math.random()*Math.random() && regex.test(message.content)) {
```
Comment (GalexY727) [inline src/events/funStuff.js:45]: Something I wanted to mention is I ran 
```js
let total = 0;
for (let i = 0; i < 10000; i+=1) {
    let res = Math.random() < Math.random()*Math.random();
    if (res) total++
}
return total/10000
```
a couple of times and it revealed that this is just a fancy way of a 25% chance. Do you think that that is too little?

Comment (PlerionPlorion) [reply src/events/funStuff.js:45]: I did some testing and it occurs between every 2-3 messages which seems better idk, honestly I just messed around till the frequency felt right.

**Pull request merged**